spec/unit/outputs/elasticsearch/http_client_spec.rb

```diff
@@ -266,6 +266,22 @@
             end
           end
         end
+
+        context "with partial unsupported action" do
+          let(:actions) { [
+            ["index", {:_id=>nil, :_index=>"logstash"}, {"message"=> "hello"}],
+            ["unsupported_action", {:_id=>nil, :_index=>"logstash"}, {"message"=> "world!"}],
+          ]}
+          it "executes one bulk_send operation" do
+            allow(subject).to receive(:join_bulk_responses)
+            expect(subject).to receive(:bulk_send).once do |body_stream, batch_actions|
```

Comment:
we can likely specify receiving `submit` instead of `bulk_send`, because `bulk_send` has its own optimizations around payload size that are out of scope.